### DISCUSSION TITLE
Create a community involvement strategy

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [alan-turing-institute/AutSPACEs#370] [Ideas]
I think it would be great to write a community involvement strategy and publish it on GitHub. I think this should include information on:

- [ ] Where/on what we need input (design, user testing, documentation feedback etc.)
- [ ] Define who is already involved and how they are involved
- [ ] Who we need to ask
- [ ] How can we reach them (which channels/dissemination and comms)
- [ ] Specific asks 
- [ ] How we use/record the feedback
- [ ] Inclusivity and accessibility checks 
- [ ] Opportunities for growth
- [ ] Metrics for success
- [ ] Understanding where our imbalance lies

What are your thoughts? It would be great to have feedback on whether to include other information, or if you have any input on any of these topics.

Thanks everyone!!

**Comment by GeorgiaHCA** [comment]
Hey @malvikasharan I'd really appreciate some of your input if you have time 🌟

**Comment by GeorgiaHCA** [comment]
Thanks for all your help @malvikasharan ! :)

I've made a Hackmd with notes and ideas on community growth

https://hackmd.io/tfZgUFHDQTqLYh-Ys9K-iw?both

**Comment by GeorgiaHCA** [comment]
https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/project-management/A9_StudyProtocol.pdf

This doc includes the protocol for running 1-2-1 user testing sessions online for which we have ethics approval

**Comment by JamesSCTJ** [comment]
Thanks Georgia. Could I add two more things:
-Metrics for success
-Understand where our imbalances lie

**Comment by GeorgiaHCA** [comment]
Heya James, yes - good idea!

**Comment by JamesSCTJ** [comment]
[Inclusivity Strategy.docx](https://github.com/alan-turing-institute/AutisticaCitizenScience/files/4583802/Inclusivity.Strategy.docx)
 Some thoughts from me on the inclusivity strategy.

**Comment by GeorgiaHCA** [comment]
Brilliant -  thank you James ! Do you mind if I put this into a markdown document and add it as a pull request for people to review? 

I agree this is an especially important point: "What other routes are there for members to participate representing the likely future shift in the projects priorities?" - in future phases of the project, we will be needing the input of people who can make changes to built environments, so finding the best ways to collaborate with them is really useful.

**Comment by GeorgiaHCA** [comment]
I wonder if you have any time next week to help brainstorm some more and pin down some actions? @jiraslama and @pgc007 and @JamesSCTJ - would really. appreciate your input too @malvikasharan if you have any time but no worries if you're too busy!

**Comment by JamesSCTJ** [comment]
> Brilliant -  thank you James ! Do you mind if I put this into a markdown document and add it as a pull request for people to review?
> I agree this is an especially important point: "What other routes are there for members to participate representing the likely future shift in the projects priorities?" - in future phases of the project, we will be needing the input of people who can make changes to built environments, so finding the best ways to collaborate with them is really useful.



> I wonder if you have any time next week to help brainstorm some more and pin down some actions? @jiraslama and @pgc007 and @JamesSCTJ - would really. appreciate your input too @malvikasharan if you have any time but no worries if you're too busy!



Absolutely to both!
J
